karaf/features/src/main/resources/resources/NOTICE

```diff
@@ -146,9 +146,15 @@ Bundled dependencies:
     License name: MIT License
     Notice:       Copyright (c) 2011-2016 Twitter, Inc.
   
-  ch.qos.logback:1.2.3: 
+  ch.qos.logback:0.1.5: 
```

Comment:
I'm curious where this version is coming from.
Brooklyn uses `1.2.9` I suppose it's a new transient dependency, but it seems very old.

It could be also a glitch in the license scripts

Reply:
Looking at mvn dependency:tree
[INFO] +- org.apache.karaf.features:framework:kar:4.3.8:compile
[INFO] |  +- org.ops4j.pax.logging:pax-logging-logback:jar:2.0.18:compile
[INFO] |  |  +- ch.qos.logback:logback-core:jar:1.2.9:compile
[INFO] |  |  +- ch.qos.logback:logback-classic:jar:1.2.9:compile
[INFO] |  |  +- ch.qos.logback.contrib:logback-jackson:jar:0.1.5:compile
[INFO] |  |  +- ch.qos.logback.contrib:logback-json-core:jar:0.1.5:compile
[INFO] |  |  \- ch.qos.logback.contrib:logback-json-classic:jar:0.1.5:compile

It seems to be a different artifact with it's own versioning

Reply:
Thanks for the clarification!